src/vs/workbench/contrib/comments/browser/commentNode.ts

```diff
@@ -518,7 +523,9 @@ export class CommentNode<T extends IRange | ICellRange> extends Disposable {
 		this._commentEditor?.layout();
 		const scrollWidth = this._body.scrollWidth;
 		const width = dom.getContentWidth(this._body);
-		this._scrollableElement.setScrollDimensions({ width, scrollWidth });
+		const scrollHeight = this._body.scrollHeight;
+		const height = dom.getContentHeight(this._body) + 4;
```

Comment:
The +4 adjustment here was found necessary to prevent a vertical scrollbar from appearing even when the content height doesn't exceed max-height. I don't understand why.